src/modules/core/components/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Avatar } from './avatar';
+
+const meta: Meta<typeof Avatar> = {
+  title: 'Core UI/Avatar',
+  component: Avatar,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof Avatar>;
+
+const URL =
+  'https://sneg.top/uploads/posts/2023-06/1688075821_sneg-top-p-avatarki-dlya-lolza-krasivo-33.jpg';
+
+export const Base: Story = {
```

Comment:
I meant for the Base use all default, it would be like this `export const Base: Story = {};`